modules/media/README.md

```diff
@@ -2,27 +2,46 @@
 
 ### 📄 Overview
 
-Media module allows users to upload, browse and edit media files associated with a specific timepoint in Loris.
+Media module allows users to **upload**, **search** and **edit** media files associated with a specific candidate timepoint in Loris.
+Any kind of data associated with a candidate timepoint can be uploaded through this module: PDFs, videos, recordings, scripts, log files, etc. Files can optionally be associated to a specific instrument form within a given candidate timepoint.
+
+
+>Note: Currently editing functionality only allows editing of certain metadata fields, such as `Comments` and `Date of Administration`.
 
 ### 🔒 Permissions
 
-In order to use media module user might need one or both of the following permissions:
+In order to use the media module the user needs one or both of the following permissions:
 
 1. **media_read** - gives user a read-only access to media module (file browsing only)
-2. **media_write** - gives user a write access to media module (file uploading, editing and deletion)
+2. **media_write** - gives user a write access to media module (upload/delete files and edit metadata)
 
 >**Note**: superusers have both of the aforementioned permissions by default! 💪
 
 ### Default upload path
 
 By default, all files are uploaded under `/data/uploads/`.
```

Comment:
Let's tell them.  e.g.  add : "(Note this directory is not created by the Loris install script and should be manually created by the admin.)"